swagger.yaml

```diff
@@ -5152,6 +5152,10 @@ definitions:
         items:
           type: object
           $ref: '#/definitions/service_network'
+      forbidden_hostnames:
```

Comment:
Add a description noting that the absence of this list implies the client (e.g. UI) should fallback to a hard-coded list, and  not that all hostnames are allowed. This will alleviate compatibility concerns of using newer UI versions with older images of the service. I'm not sure if that situation is common, but it's better to be safe